examples/monaco-graphql-nextjs/src/constants.ts

```diff
@@ -1,5 +1,5 @@
 import { editor, Uri } from 'monaco-editor';
-import { initializeMode } from 'monaco-graphql/esm/initializeMode';
+import { initializeMode } from 'monaco-graphql/dist/initializeMode';
```

Comment:
why this change? everything worked well before, so the whole js ecosystem is witching to ESM so I'll expected to keep esm imports and avoid cjs (`dist`)